.travis.yml

```diff
@@ -16,17 +16,28 @@ cache: pip
 
 jobs:
   include:
-    - name: "dependencies"
+    - name: "production dependencies"
       script:
         - pip check
+    - name: "development dependencies"
+        script:
+          - pip install -U -r requirements.txt -r requirements-dev.txt .
+          - pip check
     - name: "coverage"
       script:
+        - pip install --upgrade pip
```

Comment:
Just making sure I'm understanding this correctly: coveralls requires to have the latest pip installed?

Reply:
Not that I know of no, I wouldn't mind upgrading it, especially now when we downgrade it to the production version.. What are your thoughts?

Reply:
As we talked about IRL, to get back up to the current PIP version assuming it's both the latest and greatest